server/base/src/main/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -252,6 +252,14 @@ protected void parseConfiguration(ServerConfiguration conf) {
     tableIdToTableName = ImmutableMap.copyOf(tableIdToTableNameBuilder);
     poolNameToRegexPattern = ImmutableMap.copyOf(poolNameToRegexPatternBuilder);
 
+    LOG.info("{}", this);
```

Comment:
Is this necessary, or was it left in during development?

Reply:
It put it there because the other configuration mechanism had the same thing.  I thing it is reasonable to have a record of when the new configuration is picked up.